code/game/gamemodes/cult/talisman.dm

```diff
@@ -33,56 +17,55 @@ Rite of Disorientation
 	if(!iscultist(user))
 		user << "<span class='danger'>There are indecipherable images scrawled on the paper in what looks to be... <i>blood?</i></span>"
 		return
-	if(invocation)
+	if(invoke(user))
 		user.whisper(invocation)
-	src.invoke(user)
-	uses--
+		uses--
 	if(uses <= 0)
 		user.drop_item()
 		qdel(src)
 
-/obj/item/weapon/paper/talisman/proc/invoke(mob/living/user)
-	if(health_cost && iscarbon(user))
-		var/mob/living/carbon/C = user
-		C.apply_damage(health_cost, BRUTE, pick("l_arm", "r_arm"))
+/obj/item/weapon/paper/talisman/proc/invoke(mob/living/user, successfuluse = 1)
+	. = successfuluse
+	if(successfuluse) //if the calling whatever says we succeed, do the fancy stuff
+		if(invocation)
+			user.whisper(invocation)
+		if(health_cost && iscarbon(user))
+			var/mob/living/carbon/C = user
+			C.apply_damage(health_cost, BRUTE, pick("l_arm", "r_arm"))
 
 //Malformed Talisman: If something goes wrong.
 /obj/item/weapon/paper/talisman/malformed
 	cultist_name = "malformed talisman"
 	cultist_desc = "A talisman with gibberish scrawlings. No good can come from invoking this."
 	invocation = "Ra'sha yoka!"
 
-/obj/item/weapon/paper/talisman/malformed/invoke(mob/living/user)
+/obj/item/weapon/paper/talisman/malformed/invoke(mob/living/user, successfuluse = 1)
 	user << "<span class='cultitalic'>You feel a pain in your head. The Geometer is displeased.</span>"
 	if(iscarbon(user))
 		var/mob/living/carbon/C = user
 		C.apply_damage(10, BRUTE, "head")
 
-
 //Supply Talisman: Has a few unique effects. Granted only to starter cultists.
 /obj/item/weapon/paper/talisman/supply
 	cultist_name = "Supply Talisman"
 	cultist_desc = "A multi-use talisman that can create various objects. Intended to increase the cult's strength early on."
 	invocation = null
 	uses = 3
 
-/obj/item/weapon/paper/talisman/supply/invoke(mob/living/user)
+/obj/item/weapon/paper/talisman/supply/invoke(mob/living/user, successfuluse = 1)
 	var/dat = "<B>There are [uses] bloody runes on the parchment.</B><BR>"
 	dat += "Please choose the chant to be imbued into the fabric of reality.<BR>"
 	dat += "<HR>"
 	dat += "<A href='?src=\ref[src];rune=newtome'>N'ath reth sh'yro eth d'raggathnor!</A> - Allows you to summon an arcane tome.<BR>"
-	dat += "<A href='?src=\ref[src];rune=teleport'>Sas'so c'arta forbici!</A> - Allows you to move to a Rite of Dislocation with the keyword of \"veri\".<BR>"
+	dat += "<A href='?src=\ref[src];rune=teleport'>Sas'so c'arta forbici!</A> - Allows you to move to a selected teleportation rune.<BR>"
 	dat += "<A href='?src=\ref[src];rune=emp'>Ta'gh fara'qha fel d'amar det!</A> - Allows you to destroy technology in a short range.<BR>"
-	dat += "<A href='?src=\ref[src];rune=conceal'>Kla'atu barada nikt'o!</A> - Allows you to conceal nearby runes.<BR>"
-	dat += "<A href='?src=\ref[src];rune=reveal'>Nikt'o barada kla'atu!</A> - Allows you to reveal nearby runes.<BR>"
```

Comment:
I generally view the "supply" talisman as a source of critical starting material for the cult, not the more advanced talismans, but there's flexibility here and I'll consider adding it.